pkg/reconciler/labeler/v2/labels.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 // SyncLabels makes sure that the revisions and configurations referenced from
-// a Route are labeled with route labels.
+// a Route are labeled with route annotations.
```

Comment:
annotations or labels?

Reply:
well both. RoutingState label and the routes annotation. Naming is harder now.

Reply:
Sync/Set/Meta?

Reply:
I went through and updated them all to "routingMeta". For now I am keeping the "labeler" path/package, but when we remove v1 we might reconsider the name.